### PR TITLE
Add the output option for the delve

### PR DESCRIPTION
**Pull request by 1984weed**
Hi

I added the output option for delve. 
You would be able to use delve, even if there has been `debug` directory already.

Thank you

### CODE REVIEW

**Comment by ramya-rao-a** [comment]
@1984weed  I dont see the `--output` option when running `dlv -h` on version "1.0.0-rc.2". Is this available only on a newer version?

**Comment by 1984weed** [comment]
Sorry. It's only about `debug` mode.
https://github.com/derekparker/delve/blob/master/Documentation/usage/dlv_debug.md

**Comment by ramya-rao-a** [comment]
You will need to add the `output` entry at https://github.com/Microsoft/vscode-go/blob/master/package.json#L319 as well, else you won't get a default or intellisense when editing the launch.json file

**Comment by ramya-rao-a** [comment]
@1984weed Can you sign the CLA?

**Comment by 1984weed** [comment]
Done !!

**Comment by ramya-rao-a** [comment]
@1984weed This feature is now available in the latest update (0.6.78) to the Go extension. Thanks for the PR!

File: package.json [hunk 1]
```diff
@@ -412,6 +412,11 @@
                   "lldb"
                 ],
                 "description": "Backend used by delve. Only available in delve version 0.12.2 and above."
+              },
+              "output": {
+                "type": "string",
+                "description": "Output path for the bynary of delve",
```
Comment (levrik) [inline package.json]: Typo `binary`

Comment (ramya-rao-a) [reply package.json]: That is fixed in master

**Pull request merged**